### DISCUSSION TITLE
MoneyServer port number

### DISCUSSION THREAD

**Discussion by Unadecal** [OpenSim-NGC/OpenSim-Tranquillity#63] [Q&A]
Hi! The money server defaults to port 8008. Is there a way to assign a custom port either in MoneyServer.ini or as a command line parameter?
Many thanks.

**Comment by renevega** [comment]
Yes. In MoneyServer.ini:
[MoneyServer]
; Port No. of this Server
ServerPort = 8008

Then in OpenSim.ini refer to the new port number:
[Economy]
    CurrencyServer = "${Const|BaseURL}:8008"
